blocks/autocompleters/style.scss

```diff
@@ -2,11 +2,34 @@
 	margin-right: 8px;
 }
 
-.blocks-user-autocomplete img {
-	margin-right: 8px;
-}
-
-.blocks-user-autocomplete .slug {
-	margin-left: 8px;
-	color: $dark-gray-100;
-}
+.blocks-user-autocomplete {
+	.components-autocomplete__result img {
+		margin-right: 8px;
+		flex-grow: 0;
+		flex-shrink: 0;
+		max-width: none; // we must override the gutenberg default of 100%
+		width: 24px; // avoid jarring resize by seting the size upfront
+		height: 24px;
+	}
+	.name {
```

Comment:
Although they weren't introduced in this PR, these classnames don't adhere to Gutenberg's [BEM-like conventions](https://github.com/WordPress/gutenberg/blob/a0714e2d85e9c985a9059b45e7f984d3fc780e63/docs/coding-guidelines.md#naming). We should have something along the lines of `components-autocomplete__result-name`.

Reply:
Hmmm, reading that document I'd come up with `blocks-autocompleters__user-name` because the actual CSS is not in the `components/autocomplete` folder but the `blocks/autocompleters` folder. @mcsf  Does that sound right to you?
I would also have `blocks-autocompleters__user-slug` and `blocks-autocompleters__user-avatar`.

Reply:
I have renamed the classes. I also tweaked where the completers' className is applied in the Autocompleter structure so their css won't care about the Autocompleter's class names.